src/types.ts

```diff
@@ -357,6 +357,7 @@ export type GenomeBrowserType = {
   goto: (left: number, right: number) => void;
   jump: (location: string) => void;
   set_y: (y: number) => void;
+  switch: (path: string[], value: string[] | boolean | null) => void;
```

Comment:
The type of `value` in `switch` is any valid JSON value, see https://github.com/Ensembl/ensembl-dauphin-style-compiler/blob/3460dc08ded403007a9b026700c231f16276b791/peregrine-toolkit/src/js/jstojsonvalue.rs#L30-L41